ci-scripts/e2e-cypress.sh

```diff
@@ -93,9 +93,9 @@ else
     echo '-----'
     echo "Running Cypress end to end tests"
 
-    if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
-        yarn e2e:run:ci:core"${SUITE}"
-    else
+    # if [ "${GITHUB_EVENT_NAME}" == "pull_request" ]; then
+    #     yarn e2e:run:ci:core"${SUITE}"
+    # else
         yarn e2e:run:ci"${SUITE}"
-    fi
+    # fi
```

Comment:
Just a reminder to rollback before merging.

Reply:
will be done, thx